openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
@normanjaeckel please have a look at this. I would like to add a method like this to any of our rest models. I would like to add it as a manager instead of a classmethod but therefore we would have to add a Manager to every model we have. What do you think?

I use the method to reduce qb queries. Currently there are 2 extra queries for each(!) topic on the agenda. So at the moment there are 201 queries for the agenda list view with 100 topics.